master/docs/manual/configuration/www.rst

```diff
@@ -184,6 +184,14 @@ This server is configured with the ``www`` configuration key, which specifies a
             'Workers.showWorkerBuilders': True,
         }
 
+``ws_ping_interval``
+
+    Send websocket pings every ws_ping_interval seconds.
```

Comment:
@leoetlino : Spellcheck fails on ws_ping_interval, could you put it into backquotes so that it's interpreted as code?

Reply:
Sure. Also fixed the newsfragment